docs/src/content/docs/updates/changelog.mdx

```diff
@@ -0,0 +1,103 @@
+---
+title: "Changelog"
+description: Track API changes, feature updates, and SDK releases for the Ultravox platform.
+sidebar:
+  order: 20
+draft: true
+---
+
+import { Tabs, TabItem } from '@astrojs/starlight/components';
+import CallOut from '@components/CallOut.astro';
+
+<CallOut
+  title="Active Breaking Changes"
+  text="View current deprecation notices and migration timelines in our "
+  link={{ url: "../deprecation", text: "Deprecation Guide." }}
+/>
+
+## Quick Navigation
+- [Latest Version](#version-xyz-yyyy-mm-dd)
+- [Previous Versions](#previous-versions)
+- [All Versions](https://github.com/your-org/ultravox/releases)
+
+## Version [X.Y.Z] - YYYY-MM-DD
```

Comment:
I don't think this structure is going to work out because:

* Our REST API isn't versioned yet.
* The versions for our SDKs, our data messages API, and our REST API are unrelated to each other.

Maybe we could structure this as a chronological list of changes where each change includes a date, a description, and whatever other information is relevant for it (migration guides and deadlines for deprecated fields, links to other docs for new features, etc.).  WDYT?

Reply:
yeah agree on the issue with version numbers. was thinking that in the interim we could do as you suggest (list of things by date) then we can incorporate version numbers in future.